models/Form.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+require_once('Model.php');
+
+// Cette classe sert à manipuler tout ce qui concerne le formulaire de contact de la page d'accueil
+class Form extends Model
+{
+    //Formulaire d'accueil
+    function form_page_home_cv(){
+      if(filter_has_var(INPUT_POST, 'submit')){
+
+                $name = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'name', FILTER_SANITIZE_STRING)));
+                $email = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'email', FILTER_SANITIZE_EMAIL)));
+                $subject = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'subject', FILTER_SANITIZE_STRING)));
+                $message = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'message', FILTER_SANITIZE_STRING)));
+
+                
+                $errors = [];
+
+                if(empty($name) || empty($email) || empty($subject) || empty($message)){
+                    $errors['empty'] = "Veuillez remplier tous les champs";
+                }
+
+                if(!empty($errors)){
+                    ?>
+                        <div class="card red">
+                            <div class="card-content white-text">
+                                <?php
+                                foreach($errors as $error){
+                                    echo $error;
+                                }
+                                ?>
+                            </div>
+                        </div>
+                    <?php
+                }else{
+                    Contact::contact_mail($name,$email,$subject,$message);
+                    Contact::contact_mail_user($name,$email,$subject,$message);
+                }
+            }
+    }
+            
+    // Fonction permettant de m'envoyer un mail lorsqu'un client m'envoie un message
+    function contact_mail($name,$email,$subject,$message){
+    global $db;
+
+    $subject_mail= "Message d'un utilisateur de mon blog";
+    $message_mail = '              
+       <html lang="en" style="font-family: sans-serif;">
+            <head>
+                <meta charset="UTF-8">
+            </head>
+            <body>
+            
+                <br/>Nom de l\'utilisateur : '.$name.'
+                <br/>Adresse mail de l\'utilisateur : '.$email.'
+                <br/>Sujet du message : '.$subject.'
+                <br/>Message en entier : '.$message.'
+            </body>
+        </html>
+    ';
+
+    $header = "MIME-Version: 1.0\r\n";
+    $header .= "Content-type: text/html; charset=UTF-8\r\n";
+    $header .= 'From: jerome.lacquemant@gmail.com' . "\r\n" . 'Reply-To: jerome.lacquemant@gmail.com' . "\r\n" . 'X-Mailer: PHP/' . phpversion();
+
+    mail("jerome.lacquemant@gmail.com",$subject_mail,$message_mail,$header);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function mail() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)